boards/arm/v2m_musca/doc/v2m_musca.rst

```diff
@@ -0,0 +1,412 @@
+
+Note
+The SCC registers select the functions of pins GPIO[4:2].
+
+System Clock
+============
+
+V2M Musca has a 32.768kHz crystal clock. The clock goes to a PLL and is
+multiplied to drive the Cortex-M33 processors and SSE-200 subsystem. The
+default is 50MHz but can be increased to 170MHz maximum for the secondary
+processor (CPU1). The maximum clock frequency for the primary processor
+(CPU0) is 50MHz.
+
+Serial Port
+===========
+
+The ARM Musca processor has two UARTs. Both the UARTs have only two wires for
+RX/TX and no flow control (CTS/RTS) or FIFO. The Zephyr console output, by
+default, is utilizing UART1.
+
+Security components
+===================
+
+- Implementation Defined Attribution Unit (IDAU).
+- Secure and Non-secure configurable peripherals and memory access.
+- Secure boot.
+- Secure AMBA® interconnect
+
+Serial Configuration Controller (SCC)
+=====================================
+
+The ARM Musca test chip implements Serial Configuration Control (SCC) register.
+The purpose of this register is to allow individual control of clocks,
+reset-signals and interrupts to peripherals,as well as pin-muxing.
+
+QSPI boot memory
+================
+Normal Musca-A test chip boot operation is from external QSPI 8MB flash memory.
+Only the lowest 256KB of QSPI memory is directly accessible.
+More memory is accessible through indirect addressing.
+
+Programming and Debugging
+*************************
+
+Musca supports the v8m security extension, and by default boots to the secure
+state.
+
+When building a secure(S)/non-secure(NS application, the S one will
+have to set the idau/sau and mpc configuration to permit access from the NS,
+before jumping.
+
+The following system components are required to be properly configured during the
+secure firmware:
+
+- AHB5 TrustZone Memory Protection Controller (MPC).
+- AHB5 TrustZone Peripheral Protection Controller (PPC).
+- Implementation-Defined Attribution Unit (IDAU).
+
+For more details please refer to `Corelink SSE-200 Subsystem`_.
+
+Flashing
+========
+
+DAPLink
+---------
+
+V2M Musca provides:
+
+- A USB connection to the host computer, which exposes a Mass Storage and an
+  USB Serial Port.
+- A Serial Flash device, which implements the USB flash disk file storage.
+- A physical UART connection which is relayed over interface USB Serial port.
+
+This interfaces are exposed via DAPLink which provides:
+
+- Serial Wire Debug (SWD).
+- USB Mass Storage Device (USBMSD).
+- UART.
+- Remote reset.
+
+For more details please refer
+to `DAPLink Website`_.
+
+
+Building a secure only application
+----------------------------------
+
+
+You can build applications in the usual way. Here is an example for
+the :ref:`hello_world` application.
+
+
+Building a secure/non-secure with Zephyr on both sides
+------------------------------------------------------
+
+The process is similar to before, with the added configuration of setting-up
+the secure/non-secure configuration entry on zephyr.
+
```

Comment:
Instruction steps missing here too?

Reply:
We need to decide on how to implement Zephyr S/NS builds and how it will be expressed in the cmake system. This section will have to be updated when it is ready.

Reply:
we have basic secure only app instructions